sanic/static.py

```diff
@@ -93,6 +98,17 @@ def register(app, uri, file_or_directory, pattern,
                     headers=headers,
                     content_type=guess_type(file_path)[0] or 'text/plain')
             else:
+                if stream_large_files:
```

Comment:
Thats correct, plus it is disabled by default so existing static files performance will not have changed at all.
Are simply interested in seeing whether the `if stream_large_files:` conditional is adding any slow-down over the current `static` implementation?
I can still do the benchmark if you think it is required.